notification/app/notification/services/azure/NewsstandSender.scala

```diff
@@ -1,19 +1,40 @@
 package notification.services.azure
 
+import java.time.{Clock, Duration, Instant}
 import java.util.UUID
 
 import azure.{APNSRawPush, NotificationHubClient, Tags}
-import models.Topic
+import com.gu.notificationschedule.dynamo.{NotificationSchedulePersistenceAsync, NotificationsScheduleEntry}
+import models.{NewsstandShardConfig, Topic}
 import models.TopicTypes.Newsstand
 import notification.models.ios.NewsstandNotification
 
-class NewsstandSender(hubClient: NotificationHubClient) {
+import scala.concurrent.{ExecutionContext, Future}
 
+
+class NewsstandSender(
+                       hubClient: NotificationHubClient,
+                       legacyNewsstandRegistrationConverterConfig:NewsstandShardConfig,
+                       notificationSchedulePersistence: NotificationSchedulePersistenceAsync,
+                       clock: Clock = Clock.systemUTC()
+                     ) {
+  implicit  val executionContext:ExecutionContext = ExecutionContext.Implicits.global
```

Comment:
Given that we're in the context of a play app, I'd rather you get the execution context from play through an implicit parameter at the class level here.

Reply:
Agreed